apispec/openapi.json

```diff
@@ -4710,7 +4710,7 @@
           }
         },
         "operationId": "get-report-flow",
-        "description": "The API is to query the total traffic volume in bytes for all of customer’s transports.  Parameters allow you to specify a time period, an interval type.",
+        "description": "The API is to query the total traffic volume in bytes for all of the customer’s transports.  Parameters allow you to specify a time period and an interval type.",
```

Comment:
I suggest replacing the first sentence with "Returns the total traffic volume in bytes for all of the transports."
If our customer is calling the API, it isn't necessary or that appropriate to refer to them in the third person as "the customer" in my opinion.